upf/upf_proxy_input.c

```diff
@@ -133,26 +134,48 @@ splice_tcp_connection (flow_entry_t * flow, flow_direction_t direction)
   session_t *s;
 
   if (rev->conn_index == ~0)
-    return UPF_PROXY_INPUT_NEXT_TCP_INPUT;
+    {
+      vlib_increment_simple_counter (&gtm->upf_simple_counters
+				     [UPF_FLOW_NOT_STITCHED_NO_REV_CONNECTION],
+				     vlib_get_thread_index (), 0, 1);
+      return UPF_PROXY_INPUT_NEXT_TCP_INPUT;
+    }
 
   if (flow->dont_splice)
-    return UPF_PROXY_INPUT_NEXT_TCP_INPUT;
+    {
+      vlib_increment_simple_counter (&gtm->upf_simple_counters
+				     [UPF_FLOW_NOT_STITCHED_DONT_STITCH],
+				     vlib_get_thread_index (), 0, 1);
```

Comment:
No need to increase any counter as this will happen on each packet belonging to a flow for stitching has been rejected for any reason